Func Tests Results: Build-MS-Tx failing

When in the CLI, after creating a multi-signature account, the user is not able to build the multi-signature transaction after running: 
`pocket-core accounts build-MS-Tx 4fd7bb7c7b62752cb29163ff455ae4d31326c2d3 "{\"type\":\"pos\/Send\",\"value\":{\"from_address\":\"154fa37d8b9c6b1343620fc02abcc3bb126cfaba\",\"to_address\":\"448c23cd4f025ae76196c2b8dc5387990190474c\",\"amount\":\"1\"}}" 0b7477e98009ce1f8815a05fe5d01bcd9f14295a8d2a3e72bfd5cd215692d86d testnet 100000 true/false`

**To Reproduce**
Steps to reproduce the behavior:
1. Have the latest RC-0.6.0
2. Create properly a multisignature account.
3. Then try to build and send a multi-signature transaction by the CLI. I.e. `pocket-core accounts build-MS-Tx 4fd7bb7c7b62752cb29163ff455ae4d31326c2d3 "{\"type\":\"pos\/Send\",\"value\":{\"from_address\":\"154fa37d8b9c6b1343620fc02abcc3bb126cfaba\",\"to_address\":\"448c23cd4f025ae76196c2b8dc5387990190474c\",\"amount\":\"1\"}}" 0b7477e98009ce1f8815a05fe5d01bcd9f14295a8d2a3e72bfd5cd215692d86d testnet 100000 true/false`

**Expected behavior**
Multi-signature transaction to be build.

**Screenshots**
```
panic: reflect.Set: value of type types.MsgSend is not assignable to type types.ProtoMsg

goroutine 1 [running]:
reflect.Value.assignTo(0x117dce0, 0xc000029180, 0x199, 0x1225ab8, 0xb, 0x111ca60, 0xc000b131f0, 0xc0000f30e0, 0x7ddc0, 0xffedc0)
        /usr/local/go/src/reflect/value.go:2403 +0x432
reflect.Value.Set(0x111ca60, 0xc000b131f0, 0x194, 0x117dce0, 0xc000029180, 0x199)
        /usr/local/go/src/reflect/value.go:1537 +0xbd
github.com/tendermint/go-amino.(*Codec).decodeReflectJSONInterface(0xc0000da9a0, 0xc000b0b200, 0x80, 0x80, 0xc0000f34a0, 0x111ca60, 0xc000b131f0, 0x194, 0x0, 0x0, ...)
        /go/pkg/mod/github.com/tendermint/go-amino@v0.15.0/json-decode.go:225 +0x2fc
github.com/tendermint/go-amino.(*Codec).decodeReflectJSON(0xc0000da9a0, 0xc000aea140, 0x9d, 0xa0, 0xc0000f34a0, 0x111ca60, 0xc000b131f0, 0x194, 0x0, 0x0, ...)
        /go/pkg/mod/github.com/tendermint/go-amino@v0.15.0/json-decode.go:97 +0x9db
github.com/tendermint/go-amino.(*Codec).UnmarshalJSON(0xc0000da9a0, 0xc000aea140, 0x9d, 0xa0, 0xffedc0, 0xc000b131f0, 0x9d, 0xc000aea140)
        /go/pkg/mod/github.com/tendermint/go-amino@v0.15.0/amino.go:439 +0x1e3
github.com/pokt-network/pocket-core/codec.(*LegacyAmino).UnmarshalJSON(0xc000011098, 0xc000aea140, 0x9d, 0xa0, 0xffedc0, 0xc000b131f0, 0x0, 0x0)
        /go/src/github.com/pokt-network/pocket-core/codec/amino.go:152 +0x64
github.com/pokt-network/pocket-core/codec.(*Codec).UnmarshalJSON(...)
        /go/src/github.com/pokt-network/pocket-core/codec/codec.go:161
github.com/pokt-network/pocket-core/app.BuildMultisig(0x7ffc87e59d42, 0x28, 0x7ffc87e59d6b, 0x9d, 0xc000a93e28, 0x6, 0x7ffc87e59e4a, 0x7, 0x15f1e40, 0xc000025960, ...)
        /go/src/github.com/pokt-network/pocket-core/app/util.go:29 +0x137
github.com/pokt-network/pocket-core/app/cmd/cli.glob..func16(0x1e8bb80, 0xc00010ee40, 0x6, 0x6)
        /go/src/github.com/pokt-network/pocket-core/app/cmd/cli/accounts.go:538 +0x47a
github.com/spf13/cobra.(*Command).execute(0x1e8bb80, 0xc00010ed80, 0x6, 0x6, 0x1e8bb80, 0xc00010ed80)
        /go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x1e86cc0, 0x40731f, 0xc000088058, 0x0)
        /go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/pokt-network/pocket-core/app/cmd/cli.Execute()
        /go/src/github.com/pokt-network/pocket-core/app/cmd/cli/root.go:45 +0x2d
main.main()
        /go/src/github.com/pokt-network/pocket-core/app/cmd/pocket_core/main.go:8 +0x20
        ```

--- a/app/util.go
+++ b/app/util.go
@@ -4,12 +4,14 @@ import (
 	"encoding/base64"
 	"encoding/hex"
 	"encoding/json"
+	"log"
+	"reflect"
+
 	"github.com/pokt-network/pocket-core/crypto"
 	sdk "github.com/pokt-network/pocket-core/types"
 	"github.com/pokt-network/pocket-core/x/auth"
 	"github.com/pokt-network/pocket-core/x/auth/types"
 	pocketKeeper "github.com/pokt-network/pocket-core/x/pocketcore/keeper"
-	"log"
 )
 
 func GenerateAAT(appPubKey, clientPubKey string, key crypto.PrivateKey) (aatjson []byte, err error) {
@@ -25,10 +27,15 @@ func BuildMultisig(fromAddr, jsonMessage, passphrase, chainID string, pk crypto.
 	if err != nil {
 		return nil, err
 	}
-	var m sdk.ProtoMsg
+	var m sdk.Msg
 	if err := Codec().UnmarshalJSON([]byte(jsonMessage), &m); err != nil {
 		return nil, err
 	}
+	// use reflection to convert to proto msg
+	val := reflect.ValueOf(m)
+	vp := reflect.New(val.Type())
+	vp.Elem().Set(val)
+	protoMsg := vp.Interface().(sdk.ProtoMsg)
 	kb, err := GetKeybase()
 	if err != nil {
 		return nil, err
@@ -38,7 +45,7 @@ func BuildMultisig(fromAddr, jsonMessage, passphrase, chainID string, pk crypto.
 		auth.DefaultTxDecoder(cdc),
 		chainID,
 		"", nil).WithKeybase(kb)
-	return txBuilder.BuildAndSignMultisigTransaction(fa, pk, m, passphrase, fees, legacyCodec)
+	return txBuilder.BuildAndSignMultisigTransaction(fa, pk, protoMsg, passphrase, fees, legacyCodec)
 }
 
 func SignMultisigNext(fromAddr, txHex, passphrase, chainID string, legacyCodec bool) ([]byte, error) {

--- a/codec/codec.go
+++ b/codec/codec.go
@@ -2,6 +2,7 @@ package codec
 
 import (
 	"errors"
+
 	"github.com/gogo/protobuf/proto"
 	"github.com/pokt-network/pocket-core/codec/types"
 	tmTypes "github.com/tendermint/tendermint/types"

